app/table/QueryTable.tsx

```diff
@@ -56,6 +56,7 @@ type QueryTableProps<Item> = {
   /** Function that produces a list of actions from a row item */
   pagination?: 'inline' | 'page'
   pageSize?: number
+  rowHeight?: 'small' | 'large' | 'auto'
```

Comment:
I think you can remove `auto` as an option here and in the other one and nothing would change.